### PR TITLE
LUN-253: Change discipline item onPress behavior

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/components/DisciplineItem.tsx [hunk 1]
```diff
@@ -1,60 +1,97 @@
...
+const DescriptionContainer = styled.View`
+  flex-direction: row;
+  align-items: center;
+`
+
+const Description = styled.Text<{ pressed: boolean }>`
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  color: ${props => (props.pressed ? props.theme.colors.lunesWhite : props.theme.colors.lunesGreyMedium)};
```
Comment (klinzo) [inline src/components/DisciplineItem.tsx:51]: The color is now `lunesWhite` (`white` before). Not sure if that is correct but in this screen it is that way: https://app.zeplin.io/project/5fc7c98a435298126bcfd4ee/screen/5fd7ad2c3cfec77f5c68f10c

File: src/components/__tests__/Button.spec.tsx [hunk 2]
```diff
@@ -11,81 +11,79 @@ import resetAllMocks = jest.resetAllMocks
 
 type ButtonProps = ComponentProps<typeof Button>
 
-describe('Components', () => {
-  describe('Button', () => {
-    const onPressMock = jest.fn()
+describe('Button', () => {
```
Comment (klinzo) [inline src/components/__tests__/Button.spec.tsx:14]: Only removed the wrapping in `describe('Components', () => {...})`, same for other tests (except from `DisciplineItem.spec.tsx`).

**Comment by klinzo** [comment]
Will do this for exercises and similar in a separate PR.

**Review by f1sh1918 — APPROVED** [review]
i wonder whether there is a way to disable `pressIn` while scrolling to avoid changing background of the item but didn't find a fast solution for that.
But apart of that it works as expected.

Tested on android

**Comment by klinzo** [comment]
Please test this if this feels smooth for you.

**Review by SteffiStuffel — COMMENTED** [review]
Please rename branch (wrong prefix).

Tested, does not work as expected. I though the color should change when to blue "onPressIn" and the navigation should start when "onPressOut"?

File: src/components/DisciplineItem.tsx [hunk 3]
```diff
@@ -1,60 +1,101 @@
...
   onPress: () => void
 }
 
-const DisciplineItem = ({ selected, onPress, item, children }: DisciplineItemProps): JSX.Element => {
-  const { icon, title } = item
+const DisciplineItem = ({ onPress, icon, title, description, badgeLabel }: DisciplineItemProps): ReactElement => {
+  const [pressed, setPressed] = useState<boolean>(false)
+  const onItemPress = () => {
+    setPressed(true)
+    setTimeout(() => setPressed(false), PRESS_ANIMATION_DURATION)
```
Comment (SteffiStuffel) [inline src/components/DisciplineItem.tsx:82]: It think we should use onPressOut here.

**Comment by klinzo** [comment]
Wrong branch name, will reopen.

**Pull request closed without merging**